integrationtests/handshake/rtt.go

```diff
@@ -62,7 +62,7 @@ var _ = Describe("Handshake integration tets", func() {
 		expectedDuration := time.Duration(num) * rtt
 		Expect(testDuration).To(SatisfyAll(
 			BeNumerically(">=", expectedDuration),
-			BeNumerically("<", expectedDuration+rtt/3),
+			BeNumerically("<", expectedDuration+rtt/2),
```

Comment:
To be fair, we can probably go to `< expected duration + rtt` here, and the assertion would still be valid.